### ISSUE TITLE
aria landmark file viewer

### ISSUE THREAD

**Issue by dnoneill** [sul-dlss/sul-embed#2605]
Text not included in an ARIA landmark

All perceivable text content should be included in an [Opens in new tab:ARIA landmark](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/).

Perceivable text content refers to text that can be perceived by users of assistive technology — some of which may be invisible to sighted users.

Landmark roles provide information about page structure, and can help assistive technology users with in-page navigation. Having text outside of the landmarks will make it more difficult to navigate to for some visitors.

There are 8 standard [Opens in new tab:landmark roles](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#landmarkroles) defined in WAI-ARIA. Page sections may be given a generic region role if other ARIA landmark roles aren't appropriate.

    This is a best practice only and not currently required for WCAG conformance
    WCAG 2.2 [Opens in new tab:Technique ARIA11: Using ARIA landmarks to identify regions of a page](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA11)
    WCAG 2.2 [Opens in new tab:Technique ARIA20: Using the region role to identify a region of the page](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA20)
    Siteimprove Alfa: [Opens in new tab:Technical documentation for rule SIA-R57](https://alfa.siteimprove.com/rules/sia-r57)

Abilities affected:
Vision

![Image](https://github.com/user-attachments/assets/7e4762c1-89ba-465d-a262-30f6cf5272aa)
